lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -8889,7 +8889,7 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
         return JavascriptOperators::OP_LdModuleSlot(playout->SlotIndex1, playout->SlotIndex2, scriptContext);
     }
 
-    inline void InterpreterStackFrame::OP_StModuleSlot(Var instance, int32 slotIndex1, int32 slotIndex2, Var value)
+    inline void InterpreterStackFrame::OP_StModuleSlot(Var instance, uint32 slotIndex1, uint32 slotIndex2, Var value)
```

Comment:
>Var instance [](start = 55, length = 12)

Unused?

Reply:
Yeah.. it is borrowing the SET_ELEM_ENVSLOTNonVar in the interpreter dispatch, which to clean up will require more change.  I will do that separately.